nb2mail/__init__.py

```diff
@@ -72,9 +73,10 @@ def _file_extension_default(self):
 
     @default('template_file')
     def _template_file_default(self):
-        return 'mail.tpl'
+        # return os.path.join(os.path.dirname(__file__), "templates", "mail.tpl")
+        "mail.tpl"
```

Comment:
no return?

Reply:
weird :)
I don't understand how it ran sucessfully without that return.
Anyway I have fixed it and tested to see mails are sent sucessfully with the added `return`
Please review the file after the new commit